mappings/net/minecraft/entity/projectile/ProjectileEntity.mapping

```diff
@@ -2,6 +2,7 @@ CLASS net/minecraft/class_1676 net/minecraft/entity/projectile/ProjectileEntity
 	FIELD field_22478 ownerUuid Ljava/util/UUID;
 	FIELD field_23739 ownerEntityId I
 	FIELD field_23740 leftOwner Z
+	FIELD field_28646 hasBeenShot Z
```

Comment:
Maybe just
```suggestion
	FIELD field_28646 shot Z
```

Reply:
It's from a string but yeah it's really the same